README.md

```diff
@@ -162,6 +162,7 @@ additional details on each available environment variable.
 | `ECS_APPARMOR_CAPABLE` | `true` | Whether AppArmor is available on the container instance. | `false` | `false` |
 | `ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION` | 10m | Time to wait to delete containers for a stopped task. If set to less than 1 minute, the value is ignored.  | 3h | 3h |
 | `ECS_CONTAINER_STOP_TIMEOUT` | 10m | Time to wait for the container to exit normally before being forcibly killed. | 30s | 30s |
+| `ECS_CONTAINER_START_TIMEOUT` | 10m | Time to wait for the container to start normally before being forcibly killed. | 3m | 8m |
```

Comment:
`Time to wait for the container to start normally before being forcibly killed` is a bit misleading. How about `Timeout before giving up on starting a container`.

Reply:
OK, sounds good.